ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -172,12 +172,17 @@ StatusAppTwoPanelLayout {
 
             CommunityMembersSettingsPanel {
                 membersModel: root.community.members
+                bannedMembersModel: root.community.bannedMembers
                 editable: root.community.amISectionAdmin
                 pendingRequests: root.community.pendingRequestsToJoin ? root.community.pendingRequestsToJoin.count : 0
+                communityName: root.community.name
 
                 onUserProfileClicked: Global.openProfilePopup(id)
                 onKickUserClicked: root.rootStore.removeUserFromCommunity(id)
                 onBanUserClicked: root.rootStore.banUserFromCommunity(id)
+                onUnbanUserClicked: {
+                    root.rootStore.unbanUserFromCommunity(id)
+                }
```

Comment:
Minor esthetics: oneliners don't need brackets. It hurts my eyes when I look at lines 180-185 :sweat_smile: